test/framework/mesh/logical_volume/lv_skinmesh.lua

```diff
@@ -0,0 +1,42 @@
+--[[# test for skin (surface) mesh used as a delimiter for a logical volume
+--]]
```

Comment:
```suggestion
-- test for skin (surface) mesh used as a delimiter for a logical volume
```
Block comment for one line?